### DISCUSSION TITLE
Complex fields management

### DISCUSSION THREAD

**Discussion by cmontois** [react-hook-form/react-hook-form#4668] [Q&A] [answered]
Hey,

I have 3 fields in my front (IN / OUT / STOCK).
```
{
    a:  {
        property: ['STOCK'], // can also be IN or OUT
        b: {
              property: ['IN'], // can also be OUT or STOCK
        },
    },
    c: {
              property: ['OUT'], // can also be IN or STOCK
    },
}
```
I need to a multi select to manage IN with options (a, b, c) same for OUT and STOCK.

Currently I am using setValue to do that It works except that I don't have touchedFields do you know how I could add my fields as a touchedFields manually ?

Thanks !

**Comment by cmontois** [comment, marked as answer]
I figured that we can trigger this using set function from lodash but I don't think this is a good way of doing this :s

**Reply by bluebill1049** [reply]
that's one way, or you can schema validation.
